entrypoint.py

```diff
@@ -222,6 +237,11 @@ def run_sast(tool, command, env, config_dir, log_file=stdout, run_all=True):
         log.info(f"Skipping {tool}")
         return
 
+    if tool.lower() == "safety" :
+        if (not exists_python_code(os.getenv('PWD'))):
+            log.info("Skipping safety because there are no python project descriptors")
+            return 0
```

Comment:
What do we return when skipping a tool?

```suggestion
            return
```